### DISCUSSION TITLE
Auto collapse navbar categories when menu item is selected

### DISCUSSION THREAD

**Discussion by Babilinski** [facebook/docusaurus#5887] [Ideas]
The navbar can become messy when searching for a specific topic, it would be helpful if the navbar collapsed the other expanded categories when the user selects a menu item. This feature can be seen on other documentation websites, such as Microsoft's MRTK

**Example:**
![ezgif-2-8566d35aca23](https://user-images.githubusercontent.com/10122344/140577496-9c6779b0-1d34-4af2-9a7a-88f4f9f9e598.gif)

**Comment by Babilinski** [comment]
Created a Canny entry.
https://docusaurus.io/feature-requests/p/auto-collapse-navbar-categories-when-menu-item-is-selected

**Comment by Josh-Cena** [comment]
Is this what you have in mind? https://github.com/facebook/docusaurus/pull/3811

This is already a work in progress, I hope to complete it soon

**Comment by slorber** [comment]
IMHO this is a bad UX, as the item you click on shifts to another position just after clicking.

Others do that, but mostly because they are not Single-Page-Applications and lose sidebar state across navigation.

We can provide this as an option but not as a default.
